lansa-vmss-windows-autoscale-existing-db/README.md

```diff
@@ -0,0 +1,66 @@
+- once the scale out action is completed, the cool down period is 20 Minutes
+- if the Percent Processor Time is less than 30% for 5 Minutes, then the scale in action (remove one Virtual Machine instance) is triggered
+- once the scale in action is completed, the cool down period is 5 Minutes
+
+### Resources Deployed
++	A Virtual Network
++	Six Storage Accounts for deploying up to 100 virtual machines
++	Two public ip addresses, one for each Load Balancer
++	Two external Load Balancers
++	One Virtual Machine Scale Set to contain the single virtual machine which is responsible for configuring the database
++	One Virtual Machine Scale Set to contain the number of web servers requested by the deployer
++	The Virtual Machines are all instantiated from the Marketplace LANSA SKU lansa-scalable-license. There is a software cost for using this image. [Click here](https://azure.microsoft.com/en-us/marketplace/partners/lansa/lansa-scalable-license/) for details.
+
+## Prerequisites
+
+Before deploying this template you must:
+- Create a [LANSA supported database server](http://www.lansa.com/support/supportedversions.htm#platforms). Basic instructions are provided [here](http://docs.lansa.com/14/EN/lansa041/Content/lansa/L4WInsb4_0230.htm).
+- Construct your LANSA application using [Visual LANSA for Web Development](https://azure.microsoft.com/en-us/marketplace/partners/lansa/visuallansa/) Version 14.1 with EPCs 141010, 141011 and 141013 applied, or later.
+- Construct a deployment image MSI using the LANSA Deployment Tool provided with [Visual LANSA for Web Development](https://azure.microsoft.com/en-us/marketplace/partners/lansa/visuallansa/).
+- Upload your LANSA Web Application MSI to Azure BLOB storage and obtain the URL of the MSI. Note that the template includes a demonstration application so it is not strictly necessary to create a LANSA MSI in order to use the template.
+- Its also highly recommended to follow the usage instructions below :)
+
+## Usage Instructions
+
+For full instructions for using this template go to [Azure Deployment Tutorial](http://docs.lansa.com/14/en/lansa022/index.htm#lansa/vldtoolct_0250.htm#_Toc461606162%3FTocPath%3DLANSA%2520Application%2520Deployment%2520Tool|Cloud%2520Tutorials|Microsoft%2520Azure%2520Tutorial|_____0)
+
+
+## Notes
+
+1. Two VMSS. One to install the database; 1 to run the web site. OverProvision = false. This is so that extra VMs are not created which would put more load on the database which slows down provisioning. Failure to provision has not been seen. If it occurs, the VMSS will ensure another VM is created to establish the correct InstanceCount. A second reason is that the database installer VMSS MUST NEVER have more than 1 instance installing at a time. Errors occur when publishing the weblets. As well as the database state not being matched to what the VM thinks the state of the database is in terms of table creation, etc.
```

Comment:
Failed probe causing a VM to be terminated is a bug. That is a useful feature, but it's not one that scale sets support today. The VM is no longer sent traffic by the load balancer, but the VM doesn't get terminated.

Basically I'd like to see this sentence removed: "If it occurs, the VMSS will ensure another VM is created to establish the correct InstanceCount."